### DISCUSSION TITLE
duckdb size limit

### DISCUSSION THREAD

**Discussion by saisaigraph** [duckdb/duckdb#5469] [General]
Just wondering if there is a size limit for duckdb... Can it be 1 TB in other words can it work/handle big data scenarios...
Any help appreciated.
Thanks
Sai

**Comment by Alex-Monahan** [comment]
It can be far larger than 1 TB!
https://github.com/duckdb/duckdb/issues/1394

**Reply by saisaigraph** [reply]
Thank you Alex.
